web-ui/src/pages/PulseReportPage.jsx

```diff
@@ -307,6 +355,50 @@ const PulseReportPage = () => {
     </Card>
   );
 
+  const handleCommentClick = (pulse) => {
+    setSelectedPulse(pulse);
+    setShowComments(true);
+  };
+
+  const handleDateFromChange = dayJsDate => {
+    const date = new Date(dayJsDate.valueOf());
+    setDateFrom(date);
+    if (date > dateTo) setDateTo(date);
+  };
+
+  const handleDateToChange = dayJsDate => {
+    const date = new Date(dayJsDate.valueOf());
+    if (date < dateFrom) setDateFrom(date);
+    setDateTo(date);
+  };
+
+  const handleTeamMembersChange = members => {
+    setTeamMembers(members);
+  };
+
+  const scoreCard = (highest) => {
```

Comment:
This renders a table of either the highest five average scores or the lowest five average scores based on the boolean passed in.